model/minsky.h

```diff
@@ -151,6 +151,8 @@ namespace minsky
 
     /// write current state of all variables to the log file
     void logVariables() const;
+    
+    bool m_multipleEquities=false;
```

Comment:
What's going on here?

Reply:
It sneaked in from my develop branch. I've removed it, thanks.